.travis.yml

```diff
@@ -8,16 +8,18 @@ php:
   - hhvm
 
 env:
-  - SYMFONY_VERSION=2.5.*
+  - SYMFONY_VERSION="~2.3|~3.0"
 
 matrix:
   include:
     - php: 5.5
-      env: SYMFONY_VERSION=2.3.*
+      env: SYMFONY_VERSION=~2.3
```

Comment:
this is bad. ``~2.3`` will not run the tests on Symfony 2.3, given that it will select the latest matching version, which is 2.6 currently. The goal of this matrix is precisely to run tests against the latest Symfony 2.3 version (i.e. against the LTS)